acados/sim/sim_erk_integrator.c

```diff
@@ -748,8 +755,21 @@ int sim_erk(void *config_, sim_in *in, sim_out *out, void *opts_, void *mem_, vo
         // store hessian
         if (opts->sens_hess)
         {
-            for (i = 0; i < nhess; i++)
-                S_hess_out[i] = adj_tmp[nx + nu + i];
+            // former line for tridiagonal export was
```

Comment:
Not sure I understand what's going on here but does this code produce the same results on the experiments of the acados paper that @roversch has tested? Is there anything else left to clarify?

Reply:
yes it does in combination with the code you commented on above.